Assets/script.js

```diff
@@ -152,176 +145,208 @@ var getEventsSearch = async function (city)
     var eventSearchParams = `?apikey=${eventsAPIKey}&size=60&sort=date,name,asc`;
     eventSearchParams = !cityMod == "" ? eventSearchParams + `&city=${cityMod}` : eventSearchParams;
     //append the keyword(s) to the url if they are present
-    var evtKeyword = document.getElementById("eventsKeywordInput").value.trim();
+    var evtKeyword = document.getElementById("destinationInput").value.trim();
     eventSearchParams = !evtKeyword == "" ? eventSearchParams + `&keyword=${evtKeyword}` : eventSearchParams;
     var apiUrl = eventsAPIBaseUrl + eventSearchParams;
-    
-    try
-    {
-        //Dynamically add events to the list. The function takes: parent div, tag type, image source (if applicable), id, 
-        //id suffix, class, and text content.
-        //addEventList(parentDiv, tagType, imgSrc, id, idSuffix, classType, contentVal)
-        const res = await fetch(apiUrl);
-        const data = await res.json();
-        //max image size to display
-        var imgWidth = 205;
-        var imgHeight = 115;
-        var tagP = "<p>";
-        var newBlockRow;
- 
-        if (window.screen.height <= 900)
-        {
-            //min image size to display
-            imgWidth = 100;
-            imgHeight = 56;
-        }
+    console.log(apiUrl);
 
-        //Skip adding html elements if no event returned
-        if (data.page.totalPages >> 0)
-        {
-            for (var i = 0; i < data._embedded.events.length; i++)
-            {
-                var id = data._embedded.events[i].id;
-                newBlockRow = $("<div>")
-                //Add so row can be clickable.
-                var newLinkRow = $("<a>")
-                newLinkRow.attr("id", id);
-                
-                newLinkRow.addClass(`grid grid-cols-11 grid-rows-3 grid-flow-col gap-2 border-2 border-solid border-black`);
-                //add the event image. multiple images available so loop through to get correct size
-                for (var m = 0; m < data._embedded.events[i].images.length; m++)
-                {
-                    if (data._embedded.events[i].images[m].width == imgWidth && data._embedded.events[i].images[m].height == imgHeight)
-                    {
-                        addEventList(newLinkRow, '<div>', data._embedded.events[i].images[m].url, "row-span-3 col-span-1 max-w-full", "backgroundimage", "");
-                        break;
+    try {
+        //Dynamically add events to the list. The function takes: tag type, image source (if applicable), id, id suffix, 
+        //mouse over action, mouse out action, cursor style, class, and text content.
+        //addEventList(tagType, imgSrc, id, idSuffix, mouseOver, mouseOut, cursorStyle, classType, contentVal)
+
+        try {
```

Comment:
Why is this "try" here. It just needs the one on line 153.

Reply:
Not sure, I did not add that. We can get rid of that it looks like